.travis.yml

```diff
@@ -27,5 +27,5 @@ before_script:
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
 script:
-  - export ANDROID_NDK_HOME=/usr/local/android-sdk/ndk-bundle
-  - bash ./gradlew build connectedCheck 
+  - export ANDROID_NDK_HOME=
```

Comment:
I think you actually want 'unset ANDROID_NDK_HOME'. This just sets it to nothing.